import-export-cli/cmd/importAPI.go

```diff
@@ -808,26 +812,24 @@ func ImportAPI(credential credentials.Credential, importPath, apiImportExportEnd
 		} else {
 			fmt.Println("Existing API found, attempting to update it...")
 			fmt.Println("API ID:", id)
-			apiID = id
 			updateAPI = true
 		}
 	}
 
 	extraParams := map[string]string{}
 	httpMethod := ""
+	httpMethod = http.MethodPost
+	adminEndpoint += "/import/api"
 	if updateAPI {
-		httpMethod = http.MethodPut
```

Comment:
If the API is update where are we set the API ID?

Reply:
Or are we internally resolving it?

Reply:
we are no longer using the apiId to update the API. The backend will resolve the APIID internally using API name, version, and tenant of the loggedinuser or provider in the api.json according to preserveProvider flag value